### DISCUSSION TITLE
Unknown Command M486

### DISCUSSION THREAD

**Discussion by Fuma720** [OrcaSlicer/OrcaSlicer#3348] [Q&A] [answered]
Hi, I just started with 3D printing, and as a slicer I immediately started using Orca slicer because it seems the best to me. But in all the prints  I get the M486 error (it doesn't cause printing problems) but I was wondering what caused it and how I could resolve it.
Then another doubt I have, in some videos I saw online there were many vendors in the filaments, but I only see a few.
Sorry if I ask trivial questions but as I said I'm just starting

Some specifics:
Orca slicer v 1.9.0
Windows 10
Biqu b1
For the print settings I used the pre-set profile for the Biqu b1 and the pla biqu settings (even if I use polyterra)

**Comment by andronick83** [comment]
Disable option "Exclude Objects" if your printer does not support the "[M486](https://marlinfw.org/docs/gcode/M486.html)" command
![image](https://github.com/SoftFever/OrcaSlicer/assets/439962/ec78dac2-aaa8-4ccf-86e9-1bf1933e4a3e)

**Reply by Fuma720** [reply]
Thx for help, now i see exclude object, it was a problem of 1.9 alpha, in the 1.9 beta its fixed

**Comment by Fuma720** [comment, marked as answer]
Thanks for the reply but if I go to that section I don't see that entry, am I doing something wrong?
![image](https://github.com/SoftFever/OrcaSlicer/assets/155167919/bac127d8-6d0d-49a5-9095-4851858b8dc2)
